annotation-processing/src/integrationTest/java/com/exactpro/epfast/CreatorTests.java

```diff
@@ -1,27 +1,43 @@
 package com.exactpro.epfast;
 
-import com.exactpro.epfast.annotation.internal.CreatorImpl;
+import com.exactpro.epfast.annotation.internal.packages.com.exactpro.epfast.inside.CreatorImpl;
+import com.exactpro.epfast.inside.DefAnn;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
 
 class CreatorTests {
-    private CreatorImpl creator = new CreatorImpl();
+    private ICreator creator = CreatorService.getProviderByFastPackage("com.exactpro");
+
+    private ICreator insideCreator = CreatorService.getProviderByFastPackage("com.exactpro.epfast.inside");
+
+    @Test
+    void testGetProviderByFastPackage() {
+        assertEquals(creator.getClass(),
```

Comment:
use getFastPakage() instead. The exact class name is internal detail which we don't want to check.